src/main/java/com/github/tomakehurst/wiremock/http/ResponseDefinition.java

```diff
@@ -72,7 +75,26 @@ public ResponseDefinition(@JsonProperty("status") int status,
                               @JsonProperty("transformers") List<String> transformers,
                               @JsonProperty("transformerParameters") Parameters transformerParameters,
                               @JsonProperty("fromConfiguredStub") Boolean wasConfigured) {
-        this(status, statusMessage, Body.fromOneOf(null, body, jsonBody, base64Body), bodyFileName, headers, additionalProxyRequestHeaders, fixedDelayMilliseconds, delayDistribution, chunkedDribbleDelay, proxyBaseUrl, fault, transformers, transformerParameters, wasConfigured);
+        this(status, statusMessage, Body.fromOneOf(null, body, jsonBody, base64Body), bodyFileName, headers, additionalProxyRequestHeaders, fixedDelayMilliseconds, delayDistribution, chunkedDribbleDelay, proxyBaseUrl, fault, transformers, transformerParameters, Collections.<AbstractTransformer>emptyList(), wasConfigured);
+    }
+
+    public ResponseDefinition(int status,
```

Comment:
This constructor is a copy with an adaptation of the above as on one hand I need to pass the `transformerInstances` from the existing code usage, but on the other hand I don't want to affect JSON deserialization.